tests/settings/e2e/steps/stakepool-settings.js

```diff
@@ -0,0 +1,79 @@
+import { When, Then } from 'cucumber';
+
+const CUSTOM_SERVER_DROPDOWN_OPTION = '//*[@class="SimpleInput_customValueWrapper"]//*[@value="Custom server"]';
+const DAEDALUS_TOP_BAR_LOGO = '//*[@class="TopBar_topBar TopBar_withoutWallet"]';
+const NOT_A_VALID_SMASH_SERVER_ERROR_MESSAGE_ACTIVATED = '//*[@class="InlineEditingInput_errorMessage" and text()="This URL is not a valid SMASH server"]';
+const OFF_CHAIN_METADATA_SERVER_SMASH_LABEL = '//*[contains(text(), "Off-chain metadata server (SMASH)")]';
+const SMASH_SERVER_URL_INPUT_BOX = '//*[@label="SMASH server URL"]';
+const STAKE_POOL_CUSTOM_SERVER_INPUT_BOX_SUBMIT_BUTTON = '//*[@class="InlineEditingInput_button InlineEditingInput_okButton SimpleButton_root ButtonOverrides_root"]';
+const STAKE_POOL_CUSTOM_SERVER_INPUT_BOX_X_BUTTON = '//*[@class="InlineEditingInput_button InlineEditingInput_cancelButton SimpleButton_root ButtonOverrides_root"]';
+const STAKE_POOL_SERVER_DROPDOWN = '//*[@class="SimpleFormField_inputWrapper"]';
+const STAKE_POOL_SERVER_DROPDOWN_CUSTOM_OPTION = '//*[@class="ScrollbarsCustom-Content"]//span[text()="Custom server"]';
+const STAKE_POOLS_SUBMENU_SETTINGS = '//*[@class="SettingsMenu_component"]//button[text()="Stake pools"]';
+
+When(/^custom server is the default option$/, function() {
+  return this.waitAndClick(CUSTOM_SERVER_DROPDOWN_OPTION);
+});
+
+When(/^I clicked outside of the input-box to change focus$/, function() {
```

Comment:
Maybe there's a better solution for this. You could include more JS actions in the `I click the stake-pool custom server input box submit button` step (maybe rename it to `I focus the stake-pool custom server input box submit button`) and remove this one from the Gherkin description.

Reply:
Fixed 👍